src/github/SyftGithubAction.ts

```diff
@@ -205,14 +205,18 @@ export async function downloadSyft(): Promise<string> {
 
   await execute("sh", [installPath, "-d", "-b", syftBinaryPath, version]);
 
-  return `${syftBinaryPath}/${name}`;
+  const installedPath = path.join(`${syftBinaryPath}`, `${name}`);
+  if (process.platform === "win32") {
+    return `${installedPath}.exe`;
+  }
+  return installedPath;
 }
 
 /**
  * Gets the Syft command to run via exec
  */
 export async function getSyftCommand(): Promise<string> {
-  const name = SYFT_BINARY_NAME;
+  const name = SYFT_BINARY_NAME + (process.platform == "win32" ? ".exe" : "");
```

Comment:
nit: extract this logic to a utility called something like `getBinaryName()`, then the same function can be called above around line 208 to get the appropriate name